Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs

```diff
@@ -431,15 +427,32 @@ private void Awake()
             if (targetObject == null)
                 targetObject = gameObject;
 
+            scaleHandles = InstantiateScriptable(scaleHandles);
+            rotationHandles = InstantiateScriptable(rotationHandles);
+            boxDisplay = InstantiateScriptable(boxDisplay);
+            links = InstantiateScriptable(links);
+            proximityEffect = InstantiateScriptable(proximityEffect);
+
             // subscribe to visual changes
             scaleHandles.configurationChanged.AddListener(CreateRig);
             scaleHandles.visibilityChanged.AddListener(ResetVisuals);
             rotationHandles.configurationChanged.AddListener(CreateRig);
             boxDisplay.configurationChanged.AddListener(CreateRig);
-            links.configurationChanged.AddListener(CreateRig);
+            links.configurationChanged.AddListener(CreateRig);          
+        }
 
-            // subscribe visuals to bounds changes
-            boundsUpdated.AddListener(() => rotationHandles.CalculateEdgeCenters(ref boundsCorners));
+        private static T InstantiateScriptable<T>(T instance) where T : ScriptableObject
```

Comment:
We should move this to ScriptableObjectExtensions.

Reply:
hmm just realizing ScriptableObjectExtension is part of the Editor namespace - which would suggest those are methods that should only be used in editor mode, not runtime.